src/api/firebase.ts

```diff
@@ -25,8 +25,8 @@ export type ListPath = {
  * @returns
  */
 export const useShoppingLists = (
-	userId: string | null,
-	userEmail: string | null,
+	userId: string | null | undefined,
+	userEmail: string | null | undefined,
```

Comment:
I wonder if this might simplify a little bit

```suggestion
	userId?: string | null,
	userEmail?: string | null,
```